developer/configuration.md

```diff
@@ -25,6 +25,13 @@ For convenience, the initial Reaction administrator can be configured here.
 
 Creating a `settings.json` will prevent the default `dev.settings.json` from being loaded when you use the `reaction` command to start Reaction.
 
+Once you have edited the **_settings/settings.json_** you will need to run:
+```sh
+reaction reset && rc 
```

Comment:
I'm not sure what rc means, it's what @aaronjudd  told me to run after modifying settings.json file, and I used it and it worked.